spec/turtle.bnf

```diff
@@ -23,8 +25,8 @@ String                ::= STRING_LITERAL_QUOTE | STRING_LITERAL_SINGLE_QUOTE
 iri                   ::= IRIREF | PrefixedName
 PrefixedName          ::= PNAME_LN | PNAME_NS
 BlankNode             ::= BLANK_NODE_LABEL | ANON
-reifier               ::= '~' (iri | BlankNode)
-reifiedTriple         ::= '<<' subject predicate object reifier? '>>'
```

Comment:
Note that using `reifier*` makes this more complicated, as it can now create several triples. For example, `( << :s :p :o ~:r1 ~:r2 >> )` could result in a list with multiple elements. Everywhere else an object results in a single resource, not multiple resources.

This isn't so much of a problem with the annotation production.

We might consider going back to `'<<' ttSubject predicate ttObject reifier? '>>'`. Allowing multiple reifiers here seems to needlessly complicate the grammar and substantially change the flow of a parser.

Reply:
Let's follow-up on multiple reifiers in reifiedTriple as a follow-up.